src/managers/TileManager.ts

```diff
@@ -22,6 +22,23 @@ class TileManager {
         return this.list;
     }
 
+    /**
+     * Returns a (copy) list of every tile has TileState == Placed\
+     * If this.list has no element, return null\
+     * If this.list doesn't have any placed element, return []\
+     *
+     * @returns array of Tile or null
+     */
```

Comment:
I think i get your intention with those jsdoc string like comments but their meant for professional public libraries... also i would argue we should do it everywhere or nowhere... besides @Patertuck and i discussed it and we think the content of comments should only include things that are not easy to understand by looking at the code itself.